setup_scripts/moab_setup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+ 
+#MOAB Installation
+INSTALL_ROOT=$HOME
+cd $INSTALL_ROOT
+mkdir MOAB
+cd MOAB
+mkdir bld
+mkdir install
```

Comment:
might want a `MOAB_INSTALL_DIR` variable or similar that can be reused below to ensure consistency, especially if/when you want to change it